docs/dev-running.md

```diff
@@ -104,7 +104,7 @@ The [lightning gateway](../gateway/ln-gateway) connects the federation to the li
 $ gateway-cln balance <FEDERATION-ID>
 
 {
-  "balance_msat": 30000000
+  30000000
 }
```

Comment:
this section isn't quite right. it needs `--federation-id` before the federation id, and it just returns the balance without braces around it